### ISSUE TITLE
First Person View

### ISSUE THREAD

**Issue by dmccoystephenson** [Preponderous-Software/beyond-nations#173]
We should transition to a first-person view where the camera is controlled by the mouse.

**Comment by dmccoystephenson** [comment]
Flagging an interaction with #226, which tracks replacing Unity with a purpose-built C# engine.

This issue gets **easier** rather than harder. `UnityEngine.Camera` is used in a deliberately narrow way today — `Player` creates one, and the only property manipulated at runtime is `farClipPlane`. The replacement camera being built in #219 owns its own view and projection matrices directly, so first-person becomes a mode switch on an eye-position offset plus a toggle that skips submitting the player's own capsule to the renderer. Mouse look is part of the input layer in #217 regardless of which view is active.

Scheduling this after #219 is therefore recommended, since the work reduces to a fraction of what it would be built independently.

Regarding PR #200: the camera and mouse-look implementation there is written against `UnityEngine.Camera`, `Input.GetAxis` and `Cursor.lockState`, none of which survive the migration, so that code would need rebuilding on the new camera and input layers. What does carry over is the design decisions already settled in that PR — eye-level positioning, `A`/`D` switching from turn to strafe, hiding the player model, and cursor lock tied to the pause menu. Those are worth preserving as the specification for the reimplementation.

It should also be noted that PR #200 is stale independently of the migration. It is based on the `src/c#/` layout that `main` no longer has, currently reports as conflicting, and has not been updated since February 2026.

---

_drafted by Claude on behalf of Daniel Stephenson_

**Comment by dmccoystephenson** [comment]
Resolved in PR #250, which merged into `develop`. `V` now toggles between third- and first-person, and `--first-person` opens directly in that view. This issue stayed open because GitHub only auto-closes on merges into the default branch.

This comment was drafted during a Gardener session (https://github.com/Stephenson-Software/gardener).
